src/Loggers/LogHistograms.jl

```diff
@@ -8,8 +8,9 @@ passed as a tuple holding the `N+1` bin edges and the height of the `N` bins.
 You can also pass the raw data, and a binning algorithm from `StatsBase.jl` will
 be used to bin the data.
 """
-function log_histogram(logger::Logger, name::String, (bins,weights)::Tuple{Vector,Vector};
+function log_histogram(logger::Logger, name::String, (bins,weights)::Tuple{Vector, Array};
                        step=nothing)
+    weights = reshape(weights, :)
```

Comment:
```suggestion
    weights = vec(weights)
```

`vec` is basically an alias for `reshape(x,:)`.
(neither allocate)